pkg/nsm/apis/nsmconnect/nsmconnect.proto

```diff
@@ -12,7 +12,7 @@ message ConnectionRequest {
     string request_id = 1;
     string network_service_name = 2;
     string linux_namespace = 3;
-    repeated common.Interface interface = 4;
+    repeated common.LocalMechanism interface = 4;
```

Comment:
change interface here, too

Reply:
Yes, was going to do that subsequently (I'm going in small steps). Happy to do it in this patch if you prefer.